website/static/js/profile.js

```diff
@@ -731,6 +731,14 @@ SocialViewModel.prototype.unserialize = function(data) {
     var self = this;
     var websiteValue = [];
     $.each(data || {}, function(key, value) {
+        if (key === 'scholar') {
```

Comment:
@fabmiz You mentioned scholar specifically-- are there others beyond just scholar and profileWebsites? I imagine all should be handled.